src/main/java/io/vertx/serviceproxy/impl/InterceptorPriority.java

```diff
@@ -0,0 +1,11 @@
+package io.vertx.serviceproxy.impl;
+
+/**
+ * Defines an interceptor priority to apply them in right order
+ */
+public enum InterceptorPriority {
+
+  AUTHN,
```

Comment:
Maybe just for the sake of it, we should use the full name here?

1. Authentication
2. Authorization
3. User